src/conn/api_calc_modify.c

```diff
@@ -116,6 +116,11 @@ wiredtiger_calc_modify(WT_SESSION *wt_session,
 	const uint8_t *p1, *p2;
 	bool start;
 
+	*nentriesp = 0;
+
+	if (oldv->size == 0 || newv->size == 0)
+		return (WT_NOTFOUND);
```

Comment:
This could be extended to `if (oldv->size <= WT_CM_MINMATCH || newv->size <= WT_CM_MINMATCH)`

Reply:
Pushed in dcc612f.